.github/workflows/create_release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     outputs:
       pt_release_name: ${{ steps.release_name.outputs.pt_release_name }}
     steps:
-      - uses: malfet/checkout@silent-checkout
+      - uses: actions/checkout@v4
         with:
```

Comment:
E.g. if `quiet-checkout` was missing here in `main` we could add `show-progress: false` here and in few other places